### ISSUE TITLE
Uranium mech generator spams logs

### ISSUE THREAD

**Issue by Scinexx** [HippieStationArchive/HippieStation13-1#3064]
Like the title says, the mech generator (which is now purchasable via mining) creates radiation about every tick, spamming admin alerts in the process

**Comment by KayeArray** [comment]
I noticed this a while back when I was moaning about radiation being shitty. If you load up a mech with just uranium generators it spams the logs like a meme. It's also really shitty and releases barely enough radiation to be noteworthy.

**Comment by Scinexx** [comment]
Yeah, I got messaged by an admin because of it, I switched over to the plasma one, which seems to not be a problem.

Perhaps make it release rads in longer periods rather then every tick

**Comment by Spacedong** [comment]
MAKE IT STOP https://vgy.me/aJxp2r.png

**Comment by Spacedong** [comment]
this is urgent, someone do a hotfix
